use external Sox library instead of fetching and building at build time
Author: Jasper Grimm <jasper.grimm@york.ac.uk>
Updated for v2.3.0 by Samuel Moors (Vrije Universiteit Brussel)
--- audio-2.3.0.orig/CMakeLists.txt	2024-04-15 18:44:31.000000000 +0200
+++ audio-2.3.0/CMakeLists.txt	2024-12-16 12:54:49.496432000 +0100
@@ -167,7 +167,6 @@
 
 add_subdirectory(src/libtorchaudio)
 if (BUILD_SOX)
-  add_subdirectory(third_party/sox)
   add_subdirectory(src/libtorchaudio/sox)
 endif()
 if (USE_FFMPEG)
